tests/integrations/test_notebooks.py

```diff
@@ -10,19 +10,28 @@
 
 from unittest import TestCase
 
-def _notebook_run(path):
+default_timeout = 900
+
+
+def _notebook_run(path, timeout=default_timeout):
     """
     Execute a singular ipython notebook at path and returns the cells and outputs
 
     :returns (parsed nb object, execution errors)
     """
 
     dirname, _ = os.path.split(path)
+    execproc_timeout = '--ExecutePreprocessor.timeout=%d' % timeout
+
+    # Do not run notebook tests on Travis.  notebooks tests should only be
+    # run in the local developer testing context and notebook tests often
+    # cause time out failures on Travis builds see (github #409, #410)
+    if os.environ['TRAVIS']:
```

Comment:
This needs a modification as on the developer machine the ``os.environ['TRAVIS']`` environment variable does not exist.